lib/python/qmk/tests/test_cli_commands.py

```diff
@@ -54,3 +54,15 @@ def test_list_keyboards():
     # check to see if a known keyboard is returned
     # this will fail if handwired/onekey/pytest is removed
     assert 'handwired/onekey/pytest' in result.stdout
+
+
+def test_list_keymaps():
+    result = check_subcommand("list-keymaps", "-kb", "planck/ez")
+    assert result.returncode == 0
+    assert "planck/ez:default" and "planck/ez:drashna" in result.stdout
```

Comment:
Can we use `handwired/onekey/pytest` here instead? We can add a pytest keymap to one of the community layouts if needed.

Reply:
You are right, I've created a community layout for 'pytest'.